code/src/cljs/sixsq/nuvla/ui/intercom/spec.cljs

```diff
@@ -0,0 +1,17 @@
+(ns sixsq.nuvla.ui.intercom.spec
+  (:require-macros [sixsq.nuvla.ui.utils.spec :refer [only-keys]])
+  (:require
+    [clojure.spec.alpha :as s]))
+
+
+(s/def ::app-id any?)
+
+
+(s/def ::events any?)
+
+
+(s/def ::db (s/keys :req [::app-id ::events]))
+
+
+(def defaults {::app-id ""                                  ; "qjggmp6w"
```

Comment:
comment to be deleted? If this app-id is used it will pollute Intercom Nuvla production information. So we should create a intercom test account and use it for tests.

Reply:
Oops... this is dead code. The app-id is taken from the main/spec/intercom-api-id, without production info.